tests/t_libtest.cpp

```diff
@@ -5,7 +5,7 @@
 
 using namespace regexbench;
 
-static void report(const std::map<std::string, size_t>& m)
+void report(const std::map<std::string, size_t>& m, void* p)
```

Comment:
1. How about just keep this function static?
2. As argument `p` is not used, how about omitting `p` to suppress warning